### PR TITLE
Enable save button on page pages

### PR DESCRIPTION
**Pull request by orzechdev**
It enables save button on page pages (update and create pages), even if all required fields are not set.

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
MARKETPLACE_URL=https://marketplace-gray.vercel.app/

### Do you want to run more stable tests?
Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] giftCard
3. [ ] category
4. [ ] collection
5. [ ] attribute
6. [ ] productType
7. [ ] shipping
8. [ ] customer
9. [ ] permissions
10. [ ] menuNavigation
11. [x] pages
12. [ ] sales
13. [ ] vouchers
14. [ ] homePage
15. [ ] login
16. [ ] orders
17. [ ] products
18. [ ] app
19. [ ] plugins
20. [ ] translations

CONTAINERS=1

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-8040 Make save button enabled on Page pages](https://app.clickup.com/t/2549495/SALEOR-8040)

File: src/pages/components/PageDetailsPage/PageDetailsPage.tsx [hunk 1]
```diff
@@ -142,148 +142,156 @@ const PageDetailsPage: React.FC<PageDetailsPageProps> = ({
...
+
+        return (
+          <Container>
+            <Backlink href={pageListUrl()}>
+              {intl.formatMessage(sectionNames.pages)}
+            </Backlink>
+            <PageHeader
+              title={
+                !pageExists
+                  ? intl.formatMessage({
```
Comment (andrzejewsky) [inline src/pages/components/PageDetailsPage/PageDetailsPage.tsx:160]: let's keep translations in the messages file as in the other places, worth following that rule, consistency.

File: src/pages/components/PageDetailsPage/PageDetailsPage.tsx [hunk 2]
```diff
@@ -142,148 +142,156 @@ const PageDetailsPage: React.FC<PageDetailsPageProps> = ({
...
+                  descriptionPlaceholder={""} // TODO: Cast description to string and trim it
+                  onChange={change}
+                  slug={data.slug}
+                  slugPlaceholder={data.title}
+                  title={data.seoTitle}
+                  titlePlaceholder={data.title}
+                  helperText={intl.formatMessage({
+                    id: "jZbT0O",
+                    defaultMessage:
+                      "Add search engine title and description to make this page easier to find",
```
Comment (andrzejewsky) [inline src/pages/components/PageDetailsPage/PageDetailsPage.tsx:191]: same here, I know that perhaps you are not the author, but when you touch this, we can leave it improved : )

Comment (andrzejewsky) [reply src/pages/components/PageDetailsPage/PageDetailsPage.tsx:191]: and in each place like this in your PR

**Review by andrzejewsky — CHANGES_REQUESTED** [review]

File: src/pages/components/PageDetailsPage/PageDetailsPage.tsx (hunk 2 above)
Comment (orzechdev) [reply src/pages/components/PageDetailsPage/PageDetailsPage.tsx]: Definitely, you are right, I've updated it also.

File: src/pages/components/PageDetailsPage/PageDetailsPage.tsx [hunk 3]
```diff
@@ -142,148 +143,136 @@ const PageDetailsPage: React.FC<PageDetailsPageProps> = ({
...
-              onSubmit={attributeValues =>
-                handleAssignReferenceAttribute(attributeValues, data, handlers)
-              }
+                <CardSpacer />
+                <SeoForm
+                  errors={errors}
+                  allowEmptySlug={!pageExists}
+                  description={data.seoDescription}
+                  disabled={loading}
+                  descriptionPlaceholder={""} // TODO: Cast description to string and trim it
```
Comment (andrzejewsky) [inline src/pages/components/PageDetailsPage/PageDetailsPage.tsx:177]: ?? just noticed that - can we create a ticket for it? (maintenance / fast lane)

**Review by andrzejewsky — APPROVED** [review]

File: src/pages/components/PageDetailsPage/PageDetailsPage.tsx (hunk 3 above)
Comment (orzechdev) [reply src/pages/components/PageDetailsPage/PageDetailsPage.tsx:177]: Yep, https://app.clickup.com/t/2wguf9b

**Review by dominik-zeglen — APPROVED** [review]
I was wondering if we can use `yup` for all that validation

**Review by Droniu — APPROVED** [review]

**Pull request merged**